http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/IamProxyAuthProvider.scala

```diff
@@ -159,4 +159,14 @@ class IamProxyAuthProvider(config: Config, saProvider: ServiceAccountProvider[Fu
     creatorEmail: WorkbenchEmail,
     googleProject: GoogleProject
   )(implicit ev: ApplicativeAsk[Future, TraceId]): Future[Unit] = Future.unit
+
+  /**
+   *
+   *
+   * @param internalId
```

Comment:
I think @akarukappadath removed this class in her PR.. so this is fine, but you'll probably have to rebase.